### DISCUSSION TITLE
Upgrading Saleor to v3.6+ breaks thumbnails when using GCS storage

### DISCUSSION THREAD

**Discussion by fmravic96** [saleor/saleor#12285] [Deployment]
Hey guys,

I'm trying to upgrade Saleor to newer version (3.12) from v3.5 but I ran into a problem with thumbnail generation. I am using Google cloud storage.Thumbnails are created and displayed correctly on any version before 3.6 (before the model change).

I have read a few discussions on this topic and the solution is to set Domain url properly so it fetches the thumbnail correctly but even with that I still get 404 not found server response.

When I upload an image I get this response from the server (using Docker and Nginx):

![image](https://user-images.githubusercontent.com/28535502/224343254-ee2dee39-179b-4ead-80ce-189df336eae5.png)

Also the thumbnail isn't uploaded anywhere on the GCS platform.

Any help is greatly appreciated :)

**Comment by simon-online** [comment]
There was a breaking change to thumbnail generation that was made in v3.6. Make sure you also update the Saleor Dashboard to the latest version.
